workers/ohsome_quality_analyst/indicators/building_completeness/indicator.py

```diff
@@ -271,3 +270,17 @@ async def get_hex_cells(feature: Feature) -> FeatureCollection:
     if feature_collection["features"] is None:
         raise HexCellsNotFoundError
     return feature_collection
+
+
+async def get_shdi(featurecollection: FeatureCollection) -> list:
+    """Get SHDI values for each feature.
+
+    If feature does not intersect with the SHDI geometry take the mean of all present
+    SHDI values.
+    """
+    records = await db_client.get_shdi(featurecollection)
+    mean = np.mean([r["shdi"] for r in records])
+    shdi = [mean for feature in featurecollection.features]  # Set default
+    for r in records:
```

Comment:
to be honest, I do not really understand what you do here or why you do it this way. I think you do the following:

- get shdi values for hexgrids from database
- calculate mean
- create a new list with the length of all records and the mean shdi as value
- update the list with the actual  shdi value.

What I do not understand:
- how do you solve the problem that an shdi could be `null`
- why do you not handle this in the sql query directly? --> e.g. by setting a global default value for shdi

Reply:
@matthiasschaub please add one comment or rename `mean` to `default_value` or sth. similar. This would make the code more comprehensible :)